src/main/java/com/savvato/tribeapp/entities/NotificationType.java

```diff
@@ -7,6 +7,7 @@
 public class NotificationType {
 
     public static final NotificationType ATTRIBUTE_REQUEST_APPROVED = new NotificationType(1L, "Attribute request approved", null);
+    public static final NotificationType ATTRIBUTE_REQUEST_IN_REVIEW = new NotificationType(1L, "Attribute request in review", null);
```

Comment:
There was a bug in the previous version.. This ID should be unique, instead of 1L